CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com)
 
 ## [Unreleased]
 
+- #2032 - Fixed filter.xml on /var/acs-commons
```

Comment:
The idea is that you reference a real issue report here, not the actual PR id. AFAIK there is no issue report yet for this incorrect filter, right?

Reply:
Unless there's a hard and fast rule about this, I dont think this is a big deal.  Pretty sure this has been done before.